packages/components/transitions/src/slide-fade.tsx

```diff
@@ -125,11 +126,7 @@ export const SlideFade = forwardRef<SlideFadeProps, "div", false>(
       duration,
     }
 
-    isOpen = unmountOnExit ? isOpen && unmountOnExit : true
-
-    const css: CSSUIObject = {
-      w: "100%",
-    }
+    const isOpen = unmountOnExit ? rest.isOpen && unmountOnExit : true
```

Comment:
Why was it necessary to modify this section?If you don't need to fix it, please revert back to the original.

Also, this modification will cause a warning on the console because of the `isOpen` in `rest`.